packages/firestore/lib/FirestoreQueryModifiers.js

```diff
@@ -68,7 +68,7 @@ export default class FirestoreQueryModifiers {
   }
 
   get filters() {
-    return this._filters;
+    return this._filters.map(f => ({ ...f, fieldPath: f.fieldPath._toArray() }));
```

Comment:
Doesn't this just need to be:

`this._filters.map(f => fieldPath._toArray());`

Reply:
Everything being passed into the `_filters` array should be a `FieldPath`?

Reply:
no, the filters are the arguments passed into `where` query. So each filter looks like ->` where(FieldPath, equalityOperator, value)`

Reply:
you can actually see this below in the code: `filter.operator`, 'filter.fieldPath'. Lines 191 & 192.